src/pages/download.tsx

```diff
@@ -60,6 +60,8 @@ function MainDownload() {
                   return (<div id="otherDistributionNotice">
                         You also can install OpenRefine on Ubuntu/Debian derivatives with&nbsp;
                         <code>sudo apt install openrefine</code>
+
+                        <p>See for other systems at https://repology.org/project/openrefine/versions</p>
```

Comment:
Thank you! I think we could replace the entire message (including the lines above, which mention Ubuntu/Debian, by a link to Repology. However, I wouldn't just include the URL in plain text like you did, but instead have a clickable link. For instance:
`<p>You can also install OpenRefine via <a href="https://repology.org/project/openrefine/versions">package managers of various Linux distributions</a>.</p>`

Reply:
Done.